immutrack/src/components/login/LoginChoose.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { connect } from "react-redux";
+import { loginParent, loginProvider } from "../../actions/loginType";
+
+const LoginChoose = ({ history, loginParent, loginProvider }) => {
+  const parent = e => {
+    e.preventDefault();
+    loginParent();
+    history.push("/parent-login");
+  };
+
+  const provider = e => {
+    e.preventDefault();
+    loginProvider();
```

Comment:
Where is the return value for the loginProvider going to go?

Reply:
Same as above.